pkg/image/image.go

```diff
@@ -137,12 +137,16 @@ type Image struct {
 	Usage      Usage     `json:"usage"`
 }
 
-// AuthorizedPath checks if image is in a path supplied in paths
-func (i *Image) AuthorizedPath(paths []string) (bool, error) {
-	if err := i.initFile(); err != nil {
-		return false, err
+// Init fills in the File object if needed, because it is not passed
+//   between stages
+func (i *Image) Init() {
```

Comment:
Just can you rename this to `InitFile` or something different because there is already the `Init` function. Otherwise looks good to me

Reply:
Changed to `ReInit`